utils.h

```diff
@@ -10,7 +10,8 @@ namespace utils
 {
     // Number of milliseconds that the 'print' function waits for in-between character prints.
     int sleep_for_ms = 100;
-
+    int textsize=16;
+    int weightsize=400;
```

Comment:
'weightsize' is not a good name for this variable. also, this shouldn't be an integer, but rather a FontWeightValues enum

Reply:
Please see the naming convention followed in the rest of the files. change `textsize` to `text_size` (similar for other stuff.